internal/kafka/internal/workers/cluster_mgrs/clusters_mgr_test.go

```diff
@@ -696,15 +810,15 @@ func TestClusterManager_reconcileReadyCluster(t *testing.T) {
 		args    args
 		wantErr bool
 	}{
-		{
-			name: "should return no error if EnableReadyDataPlaneClustersReconcile is set to false",
-			fields: fields{
-				dataplaneClusterConfig: &config.DataplaneClusterConfig{
-					EnableReadyDataPlaneClustersReconcile: false,
-				},
-			},
-			wantErr: false,
-		},
+		// {
+		// 	name: "should return no error if EnableReadyDataPlaneClustersReconcile is set to false",
+		// 	fields: fields{
+		// 		dataplaneClusterConfig: &config.DataplaneClusterConfig{
+		// 			EnableReadyDataPlaneClustersReconcile: false,
+		// 		},
+		// 	},
+		// 	wantErr: false,
+		// },
 		{
```

Comment:
Should be deleted?

Reply:
I commented it out when testing. the comments should and will be removed